src/tokens/TokenDetails.tsx

```diff
@@ -116,12 +115,6 @@ export default function TokenDetailsScreen({ route }: Props) {
           />
         )}
         {token.tokenId === networkConfig.celoTokenId && <CeloNewsFeed />}
-        {token.tokenId === networkConfig.aaveArbUsdcTokenId && (
```

Comment:
I think the idea was to not have LP tokens in the tokens tab anymore, since they will show on the positions tab, I made [this ticket](https://linear.app/valora/issue/ACT-1404/dont-show-aave-pool-token-on-token-list-version-=195) as a cleanup to remove aaveArbUsdc from showing based on discussion in quick-sync.